error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1904,4 +1905,33 @@ Duration after(StepVerifier.LastStep step, Duration duration) {
       return step.verifyTimeout(duration);
     }
   }
+
+  /** Prefer {@link Mono#fromFuture(Supplier)} over {@link Mono#fromFuture(CompletableFuture)}. */
+  static final class MonoFromFuture<T> {
+    @BeforeTemplate
+    Mono<T> before(CompletableFuture<T> future) {
+      return Mono.fromFuture(future);
+    }
```

Comment:
I'll add an `XXX` to later apply `@NotMatches(IsIdentityOperation.class)`. Cause as-is this code will also rewrite `Mono.fromFuture(someVariable)`, which isn't great.